Enable VPC flow logs (#4176) (#5036)
DataBiosphere/azul: UPGRADING.rst

pushing the merge commit to the GitLab instance in that deployment.

#4176 Enable VPC flow logs
==========================

Operator
~~~~~~~~

> Manually deploy the ``gitlab`` as well as the ``shared`` component of any main

Reviewer: ```suggestion
Manually deploy the ``gitlab`` and  ``shared`` components of any main
```
